Xamarin.Forms.Build.Tasks/Xamarin.Forms.Build.Tasks.csproj

```diff
@@ -27,19 +27,16 @@
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.8.166" />
     <PackageReference Include="StrongNamer" Version="0.0.8" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
-    <PackageReference Include="GitInfo" Version="2.0.20" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\Xamarin.Forms.Core\Xamarin.Forms.Core.csproj" />
     <ProjectReference Include="..\Xamarin.Forms.Xaml\Xamarin.Forms.Xaml.csproj" />
   </ItemGroup>
-
-  <ItemGroup>
-    <Compile Include="..\Xamarin.Forms.Core\Properties\GlobalAssemblyInfo.cs">
-      <Link>Properties\GlobalAssemblyInfo.cs</Link>
-    </Compile>
-  </ItemGroup>
+  
+  <PropertyGroup>
+    <IncludeGlobalAssemblyInfo>true</IncludeGlobalAssemblyInfo>   
```

Comment:
Your projects are already SDK style. Assembly version attributes are already emitted automatically unless you explicitly disable it and the targets above that set the Version and VersionPrefix/Suffix will already integrate nicely with them. You don't need a global assembly info at all. Just remove all those `GenerateAssemblyInfo=false` (see https://github.com/xamarin/Xamarin.Forms/search?q=generateassemblyinfo&unscoped_q=generateassemblyinfo)